releases/keycloak-gatekeeper.yaml

```
@@ -0,0 +1,113 @@
+repositories:
+# No official chart, this user's chart seems to be the best there is
+- name: "gabibbo97"
+  url: "https://gabibbo97.github.io/charts/"
+
+######
+# You need to create a gatekeeper.yaml file with an array of services, with values that do not use go templating
+# Here is an example:, with comments. Most elements can be omitted
+# services:
+#    - name: dashboard  # the service name
+#      portalName: "Kubernetes Dashboard"  # The name as it should appear in the Forecastle portal (omit to hide)
+#      host: dashboard.us-west-2.corp.cpco.io # The DNS name of the public endpoint
+#      useTLS: false  # True to generate a TLS certificate and use an HTTPS endpoint
+#      upstream: https://kubernetes-dashboard.kube-system.svc.cluster.local  # The service URL
+#      skipUpstreamTlsVerify: true # True to skip the validity check of the TLS certificate of the upstream URL
+#      rules: # Array of access control rules
+#        - "uri=/*|roles=kube-admin"
+#      replicas: 1 # number of replicas
+#    - name: forecastle
+#      debug: true
+#      host: forecastle.us-west-2.corp.cpco.io
+#      upstream: http://forecastle.kube-system.svc.cluster.local
+#      rules:
+#        - "uri=/*|roles=kube-admin,user|require-any-role=true"
+######
+
+#
+environments:
+  default:
+    values:
+     - ../gatekeeper.yaml
+
+releases:
+
+#######################################################################################
+## keycloak-gatekeeper (formerly keycloak-proxy)                                     ##
+## keycloak-gatekeeper is an open source identity-aware proxy                        ##
+#######################################################################################
+
+#
+# References:
+#   - https://github.com/gabibbo97/charts/tree/master/charts/keycloak-gatekeeper
+#   - https://hub.helm.sh/charts/gabibbo97/keycloak-gatekeeper
+#   - https://www.keycloak.org/
+#
+{{ range $index, $service := .Environment.Values.services }}
+- name: "key-gate-{{- $service.name }}"
```

`gw` is a more common abbreviation for "gateway"
```suggestion
- name: "key-gw-{{- $service.name }}"
```

---

Yes, but this is not an abbreviation for "gateway", it is an abbreviation for "gatekeeper"